.gitignore

```diff
@@ -51,6 +51,7 @@
 *.sig
 
 # automatically generated source files
+**_pub_key.c
```

Comment:
The demo public key in the IAR example should still be part of the repo (it should be OK as long as the file does not get removed)